### PR TITLE
better error message from filter of API server

### PR DESCRIPTION
**Pull request by qibobo**
[#118614105]

### CODE REVIEW

File: api/src/main/java/org/cloudfoundry/autoscaler/api/filter/AuthenticationFilter.java [hunk 1]
```diff
@@ -55,7 +56,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             String authorization = httpReq.getHeader("Authorization");
             if ( authorization == null){
             	logger.info("No Authorization header found.");
-                httpResp.sendError(HttpServletResponse.SC_UNAUTHORIZED);
+                this.setResponse(httpResp, HttpServletResponse.SC_UNAUTHORIZED, MessageUtil.UNAUTHORIZED);
```
Comment (fraenkel) [inline api/src/main/java/org/cloudfoundry/autoscaler/api/filter/AuthenticationFilter.java]: This is not how MessageUtil works.
If you are going to add keys to MessageUtil, there should be a corresponding message added to the English NLS resource bundle.
You should be using the MessageUtil.getMessageXXXX method to load the appropriate message.

File: api/src/main/java/org/cloudfoundry/autoscaler/api/filter/AuthenticationFilter.java [hunk 2]
```diff
@@ -169,5 +171,14 @@ public void init(FilterConfig config) throws ServletException {
 			}
 
 	    } 
+	private void setResponse(HttpServletResponse httpResp, int status, String msg) throws IOException{
```
Comment (fraenkel) [inline api/src/main/java/org/cloudfoundry/autoscaler/api/filter/AuthenticationFilter.java]: Formatting looks wrong.

File: api/src/main/java/org/cloudfoundry/autoscaler/api/util/MessageUtil.java [hunk 3]
```diff
@@ -58,6 +58,8 @@
 	public final static String RestResponseErrorMsg_get_history_in_Get_History_context = "RestResponseErrorMsg_get_history_in_Get_History_context";
 	public final static String RestResponseErrorMsg_get_metric_in_Get_Metric_context = "RestResponseErrorMsg_get_metric_in_Get_Metric_context";
 	public final static String RestResponseErrorMsg_API_Server_retrieve_service_information_context = "RestResponseErrorMsg_API_Server_retrieve_service_information_context";
+	public final static String UNAUTHORIZED = "unauthorized";
+    public final static String INTERNAL_SERVER_ERROR = "internal_server_error";
```
Comment (fraenkel) [inline api/src/main/java/org/cloudfoundry/autoscaler/api/util/MessageUtil.java]: Formatting is wrong.

**Comment by qibobo** [comment]
@fraenkel Updated already.(1) Add message to NLS. (2) Format the code.

**Comment by fraenkel** [comment]
See tracker item for my comments.

**Pull request merged**